getResource_test.go

```diff
@@ -5,25 +5,38 @@ import (
 	"testing"
 )
 
-type fakeReturnTable []struct {
-	ID    int    `json:"id"`
-	Title string `json:"title"`
+func TestFoldersTableSlice(t *testing.T) {
+	var foldersResBody *foldersResp
+	json.Unmarshal(fakeFolderByte, &foldersResBody)
+	slice1 := []string{"707", "The Foo Folder"}
+	slice2 := []string{"708", "The Baz Folder"}
+	expectedTable := [][]string{slice1, slice2}
+	matrixTestHelper(t, foldersResBody, expectedTable)
 }
 
-func (f fakeReturnTable) TableSlice() [][]string {
-	returnSlice := []string{"200", "Test OK!"}
-	returnArray := [][]string{returnSlice}
-	return returnArray
+func TestSitesTableSlice(t *testing.T) {
+	var sitesResBody *sitesResp
+	json.Unmarshal(fakeSitesByte, &sitesResBody)
+	slice1 := []string{"1337", "Dyer"}
+	slice2 := []string{"42", "Situation"}
+	expectedTable := [][]string{slice1, slice2}
+	matrixTestHelper(t, sitesResBody, expectedTable)
 }
 
-func TestFoldersTableSlice(t *testing.T) {
-	var resBody *foldersResp
-	json.Unmarshal(fakeFolderByte, &resBody)
-	slice1 := []string{"707", "The Foo Folder"}
-	slice2 := []string{"708", "The Baz Folder"}
+func TestBrowsersTableSlice(t *testing.T) {
+	var browsersRespBody *browsersResp
+	err := json.Unmarshal(fakeClientsByte, &browsersRespBody)
+	if err != nil {
+		panic(err)
+	}
+	slice1 := []string{"firefox", "Mozilla Firefox"}
+	slice2 := []string{"ie11", "Microsoft Internet Explorer 11"}
 	expectedTable := [][]string{slice1, slice2}
-	actualTable := resBody.TableSlice()
+	matrixTestHelper(t, browsersRespBody, expectedTable)
+}
 
+func matrixTestHelper(t *testing.T, resBody returnTable, expectedTable [][]string) {
```

Comment:
Can you use [`reflect.DeepEqual()`](https://golang.org/pkg/reflect/#DeepEqual) instead of manually checking?

Also, `matrixTestHelper` isn't a very descriptive name w.r.t. what it's actually checking; I like to use "narrative" names for test functions like `checkTablesAreEqual` or something like that.